src/de/jost_net/JVerein/gui/view/FreieFormulareView.java

```diff
@@ -0,0 +1,71 @@
+package de.jost_net.JVerein.gui.view;
+
+import de.jost_net.JVerein.gui.action.DokumentationAction;
+import de.jost_net.JVerein.gui.action.MailVorlageZuweisenAction;
+import de.jost_net.JVerein.gui.control.FreieFormulareControl;
+import de.jost_net.JVerein.keys.FormularArt;
+import de.jost_net.JVerein.gui.control.FilterControl.Mitgliedstyp;
+import de.willuhn.jameica.gui.AbstractView;
+import de.willuhn.jameica.gui.GUI;
+import de.willuhn.jameica.gui.parts.Button;
+import de.willuhn.jameica.gui.parts.ButtonArea;
+import de.willuhn.jameica.gui.util.ColumnLayout;
+import de.willuhn.jameica.gui.util.LabelGroup;
+import de.willuhn.jameica.gui.util.SimpleContainer;
+
+public class FreieFormulareView extends AbstractView
+{
+
+  @Override
+  public void bind() throws Exception
+  {
+    GUI.getView().setTitle("Freie Formulare");
+
+    final FreieFormulareControl control = new FreieFormulareControl(this);
+    control.init("freieformulare.", null, null);
+
+    LabelGroup group = new LabelGroup(getParent(), "Filter");
+
+    ColumnLayout cl = new ColumnLayout(group.getComposite(), 3);
+    SimpleContainer left = new SimpleContainer(cl.getComposite());
+    left.addInput(control.getSuchAdresstyp(Mitgliedstyp.ALLE));
+    left.addInput(control.getMitgliedStatus());
+    left.addInput(control.getBeitragsgruppeAusw());
+    left.addInput(control.getMailauswahl());
+    
+    SimpleContainer mid = new SimpleContainer(cl.getComposite());
+    mid.addInput(control.getSuchname());
+    mid.addInput(control.getGeburtsdatumvon());
+    mid.addInput(control.getGeburtsdatumbis());
+    mid.addInput(control.getSuchGeschlecht());
+
+    SimpleContainer right = new SimpleContainer(cl.getComposite());
+    right.addInput(control.getEigenschaftenAuswahl());
+    right.addInput(control.getStichtag());
+    right.addInput(control.getZusatzfelderAuswahl());
+
+    SimpleContainer cont = new SimpleContainer(getParent(), true);
+    cont.addHeadline("Parameter");
+    cont.addInput(control.getAusgabeart());
+    cont.addLabelPair("Formular",
+        control.getFormular(FormularArt.FREIESFORMULAR));
+
+    cont.addHeadline("Mail");
+    cont.addInput(control.getBetreff());
+    cont.addLabelPair("Text", control.getTxt());
+
+    ButtonArea fbuttons = new ButtonArea();
+    fbuttons.addButton(control.getResetButton());
+    fbuttons.addButton(control.getSpeichernButton());
+    group.addButtonArea(fbuttons);
+
+    ButtonArea buttons = new ButtonArea();
+    buttons.addButton("Hilfe", new DokumentationAction(),
+        DokumentationUtil.KONTOAUSZUG, false, "question-circle.png");
```

Comment:
Du verweist auf die Help des Kontoauszug. Ich denke es sollte ein eigener Link sein und in der Online Help dann eine neue Seite für die Beschreibung erzeugt werden.

Reply:
Oh das hatte ich ganz vergessen, werde ich noch anlegen